### PR TITLE
Add HW 1

### PR DESCRIPTION
**Pull request by minibat4**
Add a raw solution for the homework on week 01

### CODE REVIEW

**Comment by minibat4** [comment]
Dear organizers @RiskModellingResearch, could you please verify if I have to ask our teachers Firyuza and Vlad to receive an invitation to join my repository? Amal` Ametov

**Review by RiskModellingResearch — CHANGES_REQUESTED** [review]
nn.CrossEntropyLoss() принимает на вход выход после последнего слоя logits, то есть softmax нельзя применять перед подачей в целевую функцию. Это у меня потом было поправлено)

**Comment by minibat4** [comment]
Dear @RiskModellingResearch, please verify the Softmax has been deleted, accuracy became better.

**Comment by minibat4** [comment]
@RiskModellingResearch Please verify I have improved the mistake. The error/accuracy graphs became smoother and more stable

**Review by minibat4 — COMMENTED** [review]
Looks like nothing has changed

**Review by minibat4 — COMMENTED** [review]
OK
